### ISSUE TITLE
cli has no affect?

### ISSUE THREAD

**Issue by chase-moskal** [johansatge/jpeg-autorotate#16]
hello my friend, i was happy to find this simple tool

however when i ran this on my images, even though it reported that it correctly detected which images have the botched orientation, and says that they were processed -- the images appear totally unchanged, the cli tool appears to have had no affect

here is my bash session

```
Chase@Chase-PC MINGW64 ~/Desktop/Lonnie Photoshoot 2
$ jpeg-autorotate ./*.jpg
./20190602_125845.jpg: Orientation already correct
./20190602_125957.jpg: Orientation already correct
./20190602_130009.jpg: Orientation already correct
./20190602_130014.jpg: Orientation already correct
./20190602_130026.jpg: Orientation already correct
./20190602_130101.jpg: Orientation already correct
./20190602_130104.jpg: Orientation already correct
./20190602_130230.jpg: Orientation already correct
./20190602_130231.jpg: Orientation already correct
./20190602_130233.jpg: Orientation already correct
./20190602_130235.jpg: Orientation already correct
./20190602_130239.jpg: Orientation already correct
./20190602_130242.jpg: Orientation already correct
./20190602_130243.jpg: Orientation already correct
./20190602_130246.jpg: Orientation already correct
./20190602_130247.jpg: Orientation already correct
./20190602_130250.jpg: Orientation already correct
./20190602_130252.jpg: Orientation already correct
./20190602_130253.jpg: Orientation already correct
./20190602_130254.jpg: Orientation already correct
./20190602_130255.jpg: Processed (Orientation was 6) (Quality 100%)
./20190602_130256.jpg: Processed (Orientation was 6) (Quality 100%)
```

the last two images, after the processing, appear totally unchanged

just now i have used git to confirm that not a single byte has changed

how can i get `jpeg-autorotate` to save the upright processed image?

**Comment by chase-moskal** [comment]
it looks to me like the jpeg-autorotate cli isn't writing any files, despite processing them — i can't get the cli to affect any of the files when using globs (quoted or unquoted) nor specifying the exact image path

interestingly, when using a glob, the cli outputs some handy information about each image being processed, but this information isn't available when processing a single specified image

**Comment by johansatge** [comment]
Hi! Thanks for reporting this.

Indeed, there was an issue with CLI support. This has been fixed in version `5.0.1` (pushed on npm a few minutes ago)

Let me know if you still reproduce the issue.
